test/e2e/providers/vsphere/vsphere.go

```diff
@@ -133,9 +143,321 @@ func (p *Provider) GetType() config.PlatformType {
+			Name: windowsFSSName,
+		},
+		Data: map[string]string{
+			"block-volume-snapshot":            "true",
+			"cnsmgr-suspend-create-volume":     "true",
+			"csi-auth-check":                   "true",
+			"csi-migration":                    "true",
+			"improved-csi-idempotency":         "true",
+			"improved-volume-topology":         "false",
+			"online-volume-extend":             "true",
+			"topology-preferential-datastores": "true",
+			"csi-windows-support":              "true",
+		},
+		BinaryData: nil,
+	}
+
+	// See if the ConfigMap already exists in the state we expect it to be in.
+	existingCM, err := client.CoreV1().ConfigMaps(csiNamespace).Get(context.TODO(), windowsFSSName, meta.GetOptions{})
+	if err != nil {
+		if !k8sapierrors.IsNotFound(err) {
+			return fmt.Errorf("error getting existing FSS ConfigMap: %w", err)
+		}
+	} else if err == nil {
+		if reflect.DeepEqual(existingCM.Data, fssCM.Data) {
+			// ConfigMap is already as expected, nothing to do here.
+			return nil
+		}
+		// Delete the ConfigMap as it has the wrong data.
+		err = client.CoreV1().ConfigMaps(csiNamespace).Delete(context.TODO(), windowsFSSName, meta.DeleteOptions{})
+		if err != nil {
+			return fmt.Errorf("error deleting existing FSS ConfigMap: %w", err)
+		}
+	}
+	_, err = client.CoreV1().ConfigMaps(csiNamespace).Create(context.TODO(), &fssCM, meta.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("could not create FSS ConfigMap: %w", err)
+	}
+	return nil
 }
 
-func (p *Provider) CreatePVC(_ client.Interface, _ string) (*core.PersistentVolumeClaim, error) {
-	return nil, fmt.Errorf("storage not supported on vSphere")
+// ensureWindowsCSIDaemonSet deploys the Windows CSI driver DaemonSet if it doesn't already exist
+func (p *Provider) ensureWindowsCSIDaemonSet(client client.Interface) error {
+	dsName := "vmware-vsphere-csi-driver-node-windows"
+	directoryType := core.HostPathDirectory
+	directoryOrCreate := core.HostPathDirectoryOrCreate
+	ds := apps.DaemonSet{
+		ObjectMeta: meta.ObjectMeta{
+			Name: dsName,
+		},
+		Spec: apps.DaemonSetSpec{
+			Selector: &meta.LabelSelector{MatchLabels: map[string]string{"app": dsName}},
+			Template: core.PodTemplateSpec{
+				ObjectMeta: meta.ObjectMeta{
+					Labels: map[string]string{"app": dsName},
+				},
+				Spec: core.PodSpec{
+					PriorityClassName:  "system-node-critical",
+					NodeSelector:       map[string]string{core.LabelOSStable: "windows"},
+					ServiceAccountName: "vmware-vsphere-csi-driver-node-sa",
+					OS:                 &core.PodOS{Name: core.Windows},
+					Tolerations: []core.Toleration{
+						{
+							Key:    "os",
+							Value:  "Windows",
+							Effect: core.TaintEffectNoSchedule,
+						},
+					},
+					Containers: []core.Container{
+						{
+							Name:  "node-driver-registrar",
+							Image: "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.7.0",
+							Args:  []string{"--v=5", "--csi-address=$(ADDRESS)", "-kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"},
```

Comment:
```suggestion
							Args:  []string{"--v=5", "--csi-address=$(ADDRESS)", "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"},
```

@sebsoto Should it use `--` instead? Is this picking up the correct path to the CSI driver socket?